### DISCUSSION TITLE
Marker text is not show when data size is one

### DISCUSSION THREAD

**Discussion by chaeeunDev** [patrykandpatrick/vico#897] [Questions]
### Question

Thank you for the good library.

When there are more than two data sizes, the text of the marker is visible,
The text of the marker is not visible when there is only one data.

![marker_text_now_show](https://github.com/user-attachments/assets/6a8d49b5-0e37-43ce-9c21-a452187b68bd)

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @chaeeunDev. To confirm, do you have a `Line.point` applied to your `LineCartesianLayer`?

**Reply by chaeeunDev** [reply]
yes I applied pointProvider to my chart!!

**Reply by Gowsky** [reply]
Please open a bug report. Linking this discussion is enough, so you can omit other details.

**Reply by chaeeunDev** [reply]
okay! i opened the issue #910. 
thank you for help!
